test/integration/opcache-centos8/inspec/opcache-centos8_spec.rb

```diff
@@ -0,0 +1,14 @@
+['echo "<?php phpinfo(); ?>" | php', 'curl localhost'].each do |cmd|
```

Comment:
Is this file needed?
Logic for opcache on Centos 8 is covered in centos-php72 and I don't see it referenced in the kitchen.yml